iosxr_setup.py

```diff
@@ -36,6 +36,25 @@ def open_sessions(self, username, password):
         self.xr_telnet_sessions.append(self.xr1)
 
     #
+    # Send a global-vrf XR Linux command - wait for prompt
+    # Could do bash -c or run ip netns exec global-vrf bash
+    #
+    def send_operns(self, command):
+        xr1 = self.xr1
+
+        xr1.send("bash -c %s" % command)
+        xr1.wait("[\$#]")
```

Comment:
Perhaps:

`return xr1.wait("[\$#]")`

so that you could e.g. below do

`output = send_operns("rpm -qa | grep k9sec")`